Mastodon/Scene/Profile/ProfileViewController.swift

```diff
@@ -202,33 +199,38 @@ extension ProfileViewController {
             }
             .store(in: &disposeBag)
 
-        Publishers.CombineLatest4 (
-            viewModel.relationshipViewModel.$isSuspended,
+        // build items
+        Publishers.CombineLatest4(
+            viewModel.$relationship,
             profileHeaderViewController.viewModel.$isTitleViewDisplaying,
-            editingAndUpdatingPublisher.eraseToAnyPublisher(),
-            barButtonItemHiddenPublisher.eraseToAnyPublisher()
+            editingAndUpdatingPublisher,
+            barButtonItemHiddenPublisher
         )
         .receive(on: DispatchQueue.main)
-        .sink { [weak self] isSuspended, isTitleViewDisplaying, tuple1, tuple2 in
-            guard let self = self else { return }
+        .sink { [weak self] account, isTitleViewDisplaying, tuple1, tuple2 in
+            guard let self else { return }
             let (isEditing, _) = tuple1
             let (isMeBarButtonItemsHidden, isReplyBarButtonItemHidden, isMoreMenuBarButtonItemHidden) = tuple2
 
             var items: [UIBarButtonItem] = []
             defer {
-                self.navigationItem.rightBarButtonItems = !items.isEmpty ? items : nil
```

Comment:
🥲 why not just

```
self.navigationItem.rightBarButtonItems = items.isEmpty ? nil : items
```

Isn't this short and concise enough, and maybe even get rid of the `items.isNotEmpty` extension.